src/Messages.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys */
```

Comment:
Hehe, don't disable this one :-)

Reply:
Actually, this line was supposed to be deleted ::)